src/main/java/com/minecolonies/coremod/colony/WorkManager.java

```diff
@@ -73,7 +73,10 @@ public void removeWorkOrder(final int orderId)
             final WorkOrderBuild wob = (WorkOrderBuild)workOrder;
             final AbstractBuilding building = colony.getBuilding(wob.getBuildingLocation());
             ConstructionTapeHelper.removeConstructionTape(wob, colony.getWorld());
-            building.markDirty();
+            if (building == null)
+            {
+                building.markDirty();
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') NullPointerException might be thrown as 'building' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2259)